thirdparty/xmlstream/xmlstream.cpp

```diff
@@ -2116,7 +2116,8 @@ QString XmlStreamReader::readElementText(ReadElementTextBehaviour behaviour)
                     result += readElementText(behaviour);
                     break;
                 }
-                [[gnu::fallthrough]];
+                // fall through
+                //(for ErrorOnUnexpectedElement)
```

Comment:
gcc accepts it like this, but we can just swap the lines - maybe other compilers will like it better ...